### PR TITLE
Update README file with simple demo and instruction

### PR DESCRIPTION
**Pull request by dou-du**
* Try to fix issue #21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,15 +54,15 @@ demonstrated below:
                 "lineStyle": "dash",
             },
             {
-                "label": "Co_s(up)",
+                "label": "Co (s↑)",
```
Comment (unkcpz) [inline README.md:57]: Hi @dou-du, I suggest you open a separated PR for this specifically.

**Review by unkcpz — CHANGES_REQUESTED** [review]

**Comment by unkcpz** [comment]
Thanks! @dou-du

Could you open PR for the chart.js version bump and the change to address Giovanni's suggestion in a separated PR? THis will make it is more tractable and easy to review.

File: README.md [hunk 2]
```diff
@@ -19,17 +19,76 @@ To install use pip:
...
+
+```python
+w = BandsPlotWidget(dos=dosdata, plot_fermilevel = True, show_legend = True, energy_range = {"ymin": -10.0, "ymax": 10.0})
+display(w)
+```
 
+When only plotting the density of states, the figure will be shown in
+horizontal format.
 
 For developer
```
Comment (csadorf) [inline README.md:93]:
```suggestion
For developers
```

File: README.md [hunk 3]
```diff
@@ -19,17 +19,76 @@ To install use pip:
...
+
 ```python
-w = BandsPlotWidget(bands=[banddata1, banddata2], dos=dosdata, plot_fermilevel = True, show_legend = True, energy_range = {"ymin": -13.0, "ymax": 10.0})
+w = BandsPlotWidget(bands=[banddata1, banddata2], dos=dosdata, plot_fermilevel = True, show_legend = True, energy_range = {"ymin": -10.0, "ymax": 10.0})
 display(w)
 ```
 
 In order to plot the bandstructure and density of states fiugres, one needs
 to provide band data and DOS data as json files. The examples of the input
 json files are given in the `test/data` folder. The json files for the
```
Comment (csadorf) [inline README.md:31]:
```suggestion
JSON-files are provided in the `test/data` folder. The JSON-files for the
```

File: README.md [hunk 4]
```diff
@@ -19,17 +19,76 @@ To install use pip:
 Usage
 -----
 
+### 1. Plot both bandstucture and density of states (DOS) side by side
```
Comment (csadorf) [inline README.md:22]: I'm pretty sure that this should be "band structure", compare: https://en.wikipedia.org/wiki/Electronic_band_structure
```suggestion
### 1. Plot both the band structure and the density of states (DOS) side by side
```

**Review by csadorf — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dou-du) [reply README.md:57]: Ok, I will open a separated PR for Giovanni's suggestion. However, since it is just a label here, I will not change it back.

**Comment by dou-du** [comment]
> Thanks! @dou-du
> 
> Could you open PR for the chart.js version bump and the change to address Giovanni's suggestion in a separated PR? THis will make it is more tractable and easy to review.

ok, I will do it.

File: js/package.json [hunk 5]
```diff
@@ -39,7 +39,7 @@
     "@angular/material": "10.2.2",
     "@jupyter-widgets/base": "^1.1 || ^2 || ^3 || ^4",
     "bootstrap": "3.3.7",
-    "chart.js": "2.9.4",
+    "chart.js": "3.2.0",
```
[the remark this answers is not in the document]
Comment (dou-du) [reply js/package.json:42]: I will make a separate PR for the dependencies update.

File: README.md [hunk 6]
```diff
@@ -19,17 +19,76 @@ To install use pip:
...
+
 ```python
-w = BandsPlotWidget(bands=[banddata1, banddata2], dos=dosdata, plot_fermilevel = True, show_legend = True, energy_range = {"ymin": -13.0, "ymax": 10.0})
+w = BandsPlotWidget(bands=[banddata1, banddata2], dos=dosdata, plot_fermilevel = True, show_legend = True, energy_range = {"ymin": -10.0, "ymax": 10.0})
 display(w)
 ```
 
 In order to plot the bandstructure and density of states fiugres, one needs
 to provide band data and DOS data as json files. The examples of the input
```
[the remark this answers is not in the document]
Comment (dou-du) [reply README.md:30]: Ok, I will do it.

File: js/lib/bandstructure.js [hunk 7]
```diff
@@ -37,7 +37,7 @@
 // Utility 'zip' function analogous to python's, from
 // https://stackoverflow.com/questions/4856717
 
-const Chart = require('chart.js');
+require('chart.js/dist/chart');
```
[the remark this answers is not in the document]
Comment (dou-du) [reply js/lib/bandstructure.js:40]: This new version of Chart.js needs to import differently.

[the remark this answers is not in the document]
Comment (dou-du) [reply js/lib/bandstructure.js:40]: I will change it back and make a new PR.

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (dou-du) [reply README.md]: I agree that bands data is better.

File: README.md [hunk 8]
```diff
@@ -19,19 +19,78 @@ To install use pip:
...
 display(w)
 ```
 
 In order to plot the bandstructure and density of states fiugres, one needs
-to provide band data and DOS data as json files. The examples of the input
-json files are given in the `test/data` folder. The json files for the
-bandstructure can be exported from the AiiDA program. One can import
-several bandstructure input files.
+to provide bands data and DOS data as JSON-files. The examples of the input
+JSON-files are provided in the `test/data` folder. The JSON-files for the
```
Comment (unkcpz) [inline README.md:31]:
```suggestion
JSON-files are provided in the `examples/data` folder. The JSON-files for the
```

File: README.md [hunk 9]
```diff
@@ -19,19 +19,78 @@ To install use pip:
...
+bandstructure can be exported with the [AiiDA command line interface (CLI) `verdi`](https://aiida.readthedocs.io/projects/aiida-core/en/latest/reference/command_line.html#reference-command-line) as demonstrated in
+the code below:
+
+```bash
+verdi data band export PK --format=json
```
Comment (unkcpz) [inline README.md:36]:
```suggestion
verdi data band export <PK> --format=json
```

File: README.md [hunk 10]
```diff
@@ -19,19 +19,78 @@ To install use pip:
...
+```
+
+One can plot several bandstructure input files together with the
+"widget-bandsplot". However, one can only give one input file
```
Comment (unkcpz) [inline README.md:40]:
```suggestion
widget. However, one can only give one input file
```

**Review by unkcpz — CHANGES_REQUESTED** [review]
Thanks @dou-du, just minor requests where you may forget to change. All looks good to me.

**Review by unkcpz — APPROVED** [review]
LGTM.

**Review by csadorf — COMMENTED** [review]
It seems to me that the term "bandstructure" is indeed sometimes used, but "band structure" is significantly more common. I would therefor recommend to adopt the latter. Otherwise, LGTM.

File: README.md [hunk 11]
```diff
@@ -26,17 +26,17 @@ w = BandsPlotWidget(bands=[banddata1, banddata2], dos=dosdata, plot_fermilevel =
 display(w)
 ```
 
-In order to plot the bandstructure and density of states fiugres, one needs
+In order to plot the band structure and density of states fiugres, one needs
```
Comment (csadorf) [inline README.md:29]: Typo, but I'd suggest to drop the word entirely.
```suggestion
In order to plot the band structure and density of states, one needs
```

**Review by csadorf — APPROVED** [review]
One typo, otherwise good.

**Comment by unkcpz** [comment]
Hi @dou-du, just a nitpick reminder, it will be better if you can restructure the commit message next time when merging. Thanks!

**Pull request merged**